src/main/java/seedu/address/model/event/enums/Interval.java

```diff
@@ -0,0 +1,19 @@
+package seedu.address.model.event.enums;
+
+/**
+ * Contains list of Intervals to be specified in the Calendar.
+ */
+public enum Interval {
+
+    NONE("none"),
+    DAILY("daily"),
+    WEEKLY("weekly"),
+    MONTHLY("monthly"),
+    YEARLY("yearly");
+
+    private final String value;
+
+    private Interval(String value) {
+        this.value = value;
+    }
```

Comment:
Perhaps a getter for `value` could be added, to be used in the `Recurrence` class and potentially others